src/main/resources/i18n/phrases.properties

```diff
@@ -33,3 +33,5 @@ notify.restored.success.multiple={0} items are restored
 
 status.archived=Archived
 field.preview.toolbar.status=by {0}
+
+preview.path.part1=Original location:
```

Comment:
No point in using `part1` since there's text only for one part. Change this to `preview.originalPath`